Error when converting from MV-I 2.5 SNAPSHOT

Hi, I get this error when running /pwi convert

[10:42:01 ERROR]: [PerWorldInventory] [ACF] Exception in command: perworldinventory convert 

[10:42:01 ERROR]: [PerWorldInventory] [ACF] java.lang.reflect.InvocationTargetException

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at java.lang.reflect.Method.invoke(Method.java:498)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at me.ebonjaeger.perworldinventory.command.acf.RegisteredCommand.invoke(RegisteredCommand.java:147)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at me.ebonjaeger.perworldinventory.command.acf.BaseCommand.executeCommand(BaseCommand.java:650)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at me.ebonjaeger.perworldinventory.command.acf.BaseCommand.execute(BaseCommand.java:493)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at me.ebonjaeger.perworldinventory.command.acf.RootCommand.execute(RootCommand.java:67)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at me.ebonjaeger.perworldinventory.command.acf.BukkitRootCommand.execute(BukkitRootCommand.java:62)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:729)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1761)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1565)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:5)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:12)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:109)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1016)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:943)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:841)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at java.lang.Thread.run(Thread.java:748)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] Caused by: java.lang.NoSuchMethodError: com.onarandombox.multiverseinventories.MultiverseInventories.getGroupManager()Lcom/onarandombox/multiverseinventories/api/GroupManager;

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at me.ebonjaeger.perworldinventory.conversion.ConvertService.convertPlayers(ConvertService.kt:50)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at me.ebonjaeger.perworldinventory.conversion.ConvertService.runConversion(ConvertService.kt:35)

[10:42:01 ERROR]: [PerWorldInventory] [ACF] 	at me.ebonjaeger.perworldinventory.command.ConvertCommand.onConvert(ConvertCommand.kt:40)


I have reinstalled PWI a few times since to see if it made a difference but I receive this error every time. The MV-I jar I am using is the latest dev build (https://ci.onarandombox.com/job/Multiverse-Inventories/) 

It should be noted that this server is running a recent build of Paper 1.13.2

--- a/pom.xml
+++ b/pom.xml
@@ -142,7 +142,31 @@
         <dependency>
             <groupId>com.onarandombox.multiverseinventories</groupId>
             <artifactId>Multiverse-Inventories</artifactId>
-            <version>2.5</version>
+            <version>2.5.0-SNAPSHOT</version>
+            <type>jar</type>
+            <scope>provided</scope>
+
+            <exclusions>
+                <exclusion>
+                    <groupId>org.bukkit</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+
+                <exclusion>
+                    <groupId>net.milkbowl.vault</groupId>
+                    <artifactId>Vault</artifactId>
+                </exclusion>
+
+                <exclusion>
+                    <groupId>org.codehaus.jettison</groupId>
+                    <artifactId>jettison</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>com.onarandombox.multiversecore</groupId>
+            <artifactId>Multiverse-Core</artifactId>
+            <version>2.5.0-SNAPSHOT</version>
             <type>jar</type>
             <scope>provided</scope>
 

--- a/src/main/kotlin/me/ebonjaeger/perworldinventory/command/ConvertCommand.kt
+++ b/src/main/kotlin/me/ebonjaeger/perworldinventory/command/ConvertCommand.kt
@@ -24,16 +24,15 @@ class ConvertCommand @Inject constructor(private val pluginManager: PluginManage
     {
         if (!pluginManager.isPluginEnabled("Multiverse-Inventories"))
         {
-            sender.sendMessage("${ChatColor.DARK_RED}» ${ChatColor.GRAY}" +
-                    "Multiverse-Inventories is not installed! Cannot convert data!")
+            sender.sendMessage(
+                    "${ChatColor.DARK_RED}» ${ChatColor.GRAY}Multiverse-Inventories is not installed! Cannot convert data!")
             return
         }
 
         val mvi = pluginManager.getPlugin("Multiverse-Inventories")
         if (mvi == null)
         {
-            sender.sendMessage("${ChatColor.DARK_RED}» ${ChatColor.GRAY}" +
-                    "Unable to get Multiverse-Inventories instance!")
+            sender.sendMessage("${ChatColor.DARK_RED}» ${ChatColor.GRAY}Unable to get Multiverse-Inventories instance!")
             return
         }
 

--- a/src/main/kotlin/me/ebonjaeger/perworldinventory/conversion/ConvertExecutor.kt
+++ b/src/main/kotlin/me/ebonjaeger/perworldinventory/conversion/ConvertExecutor.kt
@@ -1,10 +1,10 @@
 package me.ebonjaeger.perworldinventory.conversion
 
 import ch.jalu.injector.annotations.NoMethodScan
-import com.onarandombox.multiverseinventories.ProfileTypes
-import com.onarandombox.multiverseinventories.api.profile.PlayerProfile
-import com.onarandombox.multiverseinventories.api.profile.WorldGroupProfile
-import com.onarandombox.multiverseinventories.api.share.Sharables
+import com.onarandombox.multiverseinventories.WorldGroup
+import com.onarandombox.multiverseinventories.profile.PlayerProfile
+import com.onarandombox.multiverseinventories.profile.ProfileTypes
+import com.onarandombox.multiverseinventories.share.Sharables
 import me.ebonjaeger.perworldinventory.ConsoleLogger
 import me.ebonjaeger.perworldinventory.Group
 import me.ebonjaeger.perworldinventory.GroupManager
@@ -30,7 +30,7 @@ class ConvertExecutor @Inject constructor(private val groupManager: GroupManager
                                           @DataDirectory private val dataDirectory: File)
 {
 
-    var mvGroups: List<WorldGroupProfile>? = null
+    var mvGroups: List<WorldGroup>? = null
 
     /**
      * Converts data from the MultiVerse-Inventories format to the PWI format.
@@ -62,7 +62,7 @@ class ConvertExecutor @Inject constructor(private val groupManager: GroupManager
 
                 try
                 {
-                    val playerProfile = mvGroup.getPlayerData(type, player)
+                    val playerProfile = mvGroup.groupProfileContainer.getPlayerData(type, player)
                     if (playerProfile != null)
                     {
                         val data = convertData(playerProfile)

--- a/src/main/kotlin/me/ebonjaeger/perworldinventory/conversion/ConvertService.kt
+++ b/src/main/kotlin/me/ebonjaeger/perworldinventory/conversion/ConvertService.kt
@@ -2,12 +2,14 @@ package me.ebonjaeger.perworldinventory.conversion
 
 import ch.jalu.injector.annotations.NoMethodScan
 import com.onarandombox.multiverseinventories.MultiverseInventories
+import me.ebonjaeger.perworldinventory.ConsoleLogger
 import me.ebonjaeger.perworldinventory.GroupManager
 import me.ebonjaeger.perworldinventory.service.BukkitService
 import org.bukkit.ChatColor
 import org.bukkit.GameMode
 import org.bukkit.OfflinePlayer
 import org.bukkit.command.CommandSender
+import org.bukkit.command.ConsoleCommandSender
 import org.bukkit.plugin.PluginManager
 import javax.inject.Inject
 
@@ -35,16 +37,20 @@ class ConvertService @Inject constructor(private val bukkitService: BukkitServic
         convertPlayers(sender, offlinePlayers, mvinventory)
     }
 
-    private fun convertPlayers(sender: CommandSender,
-                               offlinePlayers: Array<out OfflinePlayer>,
+    private fun convertPlayers(sender: CommandSender, offlinePlayers: Array<out OfflinePlayer>,
                                mvinventory: MultiverseInventories)
     {
         if (converting)
         {
-            sender.sendMessage("${ChatColor.DARK_RED}» ${ChatColor.GRAY}" +
-                    "A conversion is already in progress!")
+            sender.sendMessage("${ChatColor.DARK_RED}» ${ChatColor.GRAY}A conversion is already in progress!")
             return
         }
+
+        if (sender !is ConsoleCommandSender) {
+            ConsoleLogger.info(
+                    "${ChatColor.BLUE}» ${ChatColor.GRAY}Beginning conversion from Multiverse-Inventories...")
+        }
+        sender.sendMessage("${ChatColor.BLUE}» ${ChatColor.GRAY}Beginning conversion from Multiverse-Inventories...")
 
         converting = true
         val mvGroups = mvinventory.groupManager.groups

--- a/src/main/kotlin/me/ebonjaeger/perworldinventory/conversion/ConvertTask.kt
+++ b/src/main/kotlin/me/ebonjaeger/perworldinventory/conversion/ConvertTask.kt
@@ -1,8 +1,11 @@
 package me.ebonjaeger.perworldinventory.conversion
 
+import me.ebonjaeger.perworldinventory.ConsoleLogger
 import org.bukkit.ChatColor
 import org.bukkit.OfflinePlayer
 import org.bukkit.command.CommandSender
+import org.bukkit.command.ConsoleCommandSender
+import org.bukkit.entity.Player
 import org.bukkit.scheduler.BukkitRunnable
 
 /**
@@ -44,8 +47,8 @@ class ConvertTask (private val convertService: ConvertService,
         convertService.executeConvert(playersInPage)
         if (currentPage % 20 == 0)
         {
-            sender.sendMessage("${ChatColor.BLUE}» ${ChatColor.GRAY}Convert progress: " +
-                    "$stopIndex/${offlinePlayers.size}")
+            ConsoleLogger.info(
+                    "${ChatColor.BLUE}» ${ChatColor.GRAY}Convert progress: $stopIndex/${offlinePlayers.size}")
         }
     }
 
@@ -53,13 +56,16 @@ class ConvertTask (private val convertService: ConvertService,
     {
         cancel()
 
-        sender.sendMessage("${ChatColor.BLUE}» ${ChatColor.GRAY}Conversion has " +
-                "been completed! Disabling MultiVerse-Inventories...")
+        if (sender !is ConsoleCommandSender) {
+            ConsoleLogger.info(
+                    "${ChatColor.BLUE}» ${ChatColor.GRAY}Conversion has been completed! Disabling MultiVerse-Inventories!")
+        }
+        if (sender is Player && sender.isOnline) {
+            sender.sendMessage(
+                    "${ChatColor.BLUE}» ${ChatColor.GRAY}Conversion has been completed! Disabling MultiVerse-Inventories!")
+        }
 
         convertService.finish()
-
-        sender.sendMessage("${ChatColor.BLUE}» ${ChatColor.GRAY}" +
-                "MultiVerse-Inventories disabled! Don't forget to remove the .jar!")
 
         convertService.converting = false
     }

--- a/src/test/kotlin/me/ebonjaeger/perworldinventory/conversion/ConvertTaskTest.kt
+++ b/src/test/kotlin/me/ebonjaeger/perworldinventory/conversion/ConvertTaskTest.kt
@@ -7,7 +7,7 @@ import me.ebonjaeger.perworldinventory.ReflectionUtils
 import org.bukkit.Bukkit
 import org.bukkit.OfflinePlayer
 import org.bukkit.Server
-import org.bukkit.command.CommandSender
+import org.bukkit.entity.Player
 import org.bukkit.scheduler.BukkitRunnable
 import org.bukkit.scheduler.BukkitScheduler
 import org.bukkit.scheduler.BukkitTask
@@ -37,7 +37,7 @@ class ConvertTaskTest
                 mock(OfflinePlayer::class.java), mock(OfflinePlayer::class.java), mock(OfflinePlayer::class.java),
                 mock(OfflinePlayer::class.java), mock(OfflinePlayer::class.java), mock(OfflinePlayer::class.java))
 
-        val task = ConvertTask(convertService, mock(CommandSender::class.java), players)
+        val task = ConvertTask(convertService, mock(Player::class.java), players)
 
         // when (1 - first run, 5 players per run)
         task.run()
@@ -59,7 +59,8 @@ class ConvertTaskTest
     fun shouldStopAndInformOnComplete()
     {
         // given
-        val sender = mock(CommandSender::class.java)
+        val sender = mock(Player::class.java)
+        given(sender.isOnline).willReturn(true)
         val players = emptyArray<OfflinePlayer>()
         val task = ConvertTask(convertService, sender, players)
         val bTask = mock(BukkitTask::class.java)

